p2p/types/router.go

```diff
@@ -9,11 +9,11 @@ import (
 
 // TECHDEBT(olshansky): When we delete `stdnetwork` and only go with `raintree`, this interface
 // can be simplified greatly.
-type Network interface {
+type Router interface {
```

Comment:
Very much agree that Router is better than Network. It's clearer what it's actually doing in the code.